server/neptune/adhoc/adhocexecutionhelpers/adhoc_execution_params.go

```diff
@@ -16,15 +19,29 @@ type AdhocTerraformWorkflowExecutionParams struct {
 	// Note that deploymentID is used in NewWorkflowStore(), but we don't care about that in adhoc mode so can leave it blank
 }
 
-func ConstructAdhocExecParamsWithRootCfgBuilderAndRepoRetriever(ctx context.Context, repoName string, revision string, githubRetriever *adhocgithubhelpers.AdhocGithubRetriever) (AdhocTerraformWorkflowExecutionParams, error) {
+func ConstructAdhocExecParamsWithRootCfgBuilderAndRepoRetriever(ctx context.Context, repoName string, revision string, githubRetriever *adhocgithubhelpers.AdhocGithubRetriever, rootCfgBuilder *root_config.Builder) (AdhocTerraformWorkflowExecutionParams, error) {
 	// TODO: in the future, could potentially pass in the owner instead of hardcoding lyft
 	repo, err := githubRetriever.GetRepository(ctx, "lyft", repoName)
 	if err != nil {
 		return AdhocTerraformWorkflowExecutionParams{}, errors.Wrap(err, "getting repo")
 	}
 
+	opts := config.BuilderOptions{
+		RepoFetcherOptions: &internal_gh.RepoFetcherOptions{
+			CloneDepth: 1,
```

Comment:
It's not required, but do you want to enable SimplePath here?

Reply:
oh yeah I forgot